Civilla-Project/Police-Control/src/main/java/CommandTrees.java

```diff
@@ -0,0 +1,34 @@
+import java.util.ArrayList;
+
+public class CommandTrees {
+    static BotCommand basicTree =
+            new BotCommand("base","Hello there, citizen! It is civilla system. What do you want to do?", new GoToLabel(), new String[]{"Edit my data", "View my data", "About us"}, new BotCommand[]{
```

Comment:
There is a lot of hard-coded string here and in other files. Maybe move it to resources?